tests/unit/torch/quantization/plugins/test_pytorch_geometric_plugin.py

```diff
@@ -0,0 +1,178 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for PyTorch Geometric quantization plugin."""
+
+import pytest
+import torch
+import torch.nn as nn
+
+import modelopt.torch.quantization as mtq
+
+torch_geometric = pytest.importorskip(
+    "torch_geometric", reason="PyTorch Geometric is not installed"
+)
```

Comment:
Can you include `torch_geometric` dependency under `setup.py` for `dev-test` section so this unit test runs in CICD regularly.

https://github.com/NVIDIA/TensorRT-Model-Optimizer/blob/main/setup.py#L75